common/testData/units/unitPreferencesTest.txt

```diff
@@ -2,7 +2,7 @@
 # Test data for unit preferences
```

Comment:
Add a comment, "DO NOT EDIT THIS FILE, it is autogenerated by GenerateUnitTestData.java"?